Makefile

```diff
@@ -13,20 +13,20 @@ help:  ## Prints all the targets in all the Makefiles
 list:  ## List all make targets
 	@${MAKE} -pRrn : -f $(MAKEFILE_LIST) 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | egrep -v -e '^[^[:alnum:]]' -e '^$@$$' | sort
 
+#####################
+### Documentation ###
+#####################
+
 ## Ensure godoc is installed
 .PHONY: check_godoc
-# Internal helper target - check if godoc is installed
+	# Internal helper target - check if godoc is installed
 check_godoc:
 	{ \
-	if ( ! ( command -v godoc >/dev/null )); then \
+		if ( ! ( command -v godoc >/dev/null )); then \
 		echo "Seems like you don't have godoc installed. Make sure you install it via 'go install golang.org/x/tools/cmd/godoc@latest' before continuing"; \
 		exit 1; \
-	fi; \
-	}
-
-#####################
-### Documentation ###
-#####################
+		fi; \
```

Comment:
Was this intentional?

Reply:
Yeah this level of intentation is required my Makefile syntax its anoying